resultsdbupdater/consumer.py

```diff
@@ -43,3 +43,5 @@ def consume(self, msg):
             if result_keys and result_keys[0].startswith('dva.ami'):
                 self.debug_log_msg(msg)
                 return resultsdb_post_to_resultsdb(msg)
+        else:
+            LOGGER.warn("Received unhandled message topic %r" % msg['topic'])
```

Comment:
This will be very noisy because we ignore almost all messages from the "/topic/VirtualTopic.qe.ci.jenkins'" topic.

Reply:
Ah, but those will be caught in the conditional at line 39.

This will only log for messages that aren't handled in *any* of the topic conditions above.

Reply:
Oops, you're right.